### ISSUE TITLE
data elements collected and not used

### ISSUE THREAD

**Issue by vojtechhuser** [CBIIT/FHH#11]
To increase usability, if a data element is not used in the calculations, it should perhaps not be collected.

The site collects Askenazi Jew origin, but it is not used by the model.

**Comment by bobnih** [comment]
Vojtech, are you guys still in session? I can walk over there in about 20 minutes…
 Bob

-- Bob Wildin, M.D.  GHB/DPCE/NHGRI/NIH  Direct Tel: 301-451-0295

**Comment by bobnih** [comment]
This was probably done in anticipation of adding a BRCA model, where the Ashkenazi background highlights particular founder mutations with slightly different characteristics, and may allow targeted testing that saves time/money.
Since we now have a partially built breast cancer module, we may not want to remove that question from the user interface just yet. – Bob Wildin

**Comment by bremonline** [comment]
Yes, there are disease risk calculators that take into account Ashkenazi heritage.  We have not implemented it yet.
